core/sr-primitives/src/sr_arithmetic.rs

```diff
@@ -178,7 +178,7 @@ macro_rules! implement_per_thing {
 					// `rem_multiplied_upper` is less than $max^2 therefore divided by $max it fits
 					// in $type. remember that $type always fits $max.
 					let mut rem_multiplied_divided_sized =
-						(rem_multiplied_upper / upper_max) as $type;
+						rem_multiplied_upper.div(upper_max) as $type;
```

Comment:
Hmm, how is that better?

Reply:
It is just to avoid `https://rust-lang.github.io/rust-clippy/master/index.html#suspicious_arithmetic_impl`. Probably a false-positive

Reply:
Could you please revert this? We should not include fixes for false positives in clippy.